can_tools/scrapers/base.py

```diff
@@ -402,6 +440,26 @@ def validate(self, df, df_hist):
         validated : bool
             Whether we have validated the data
         """
+        if is_time_series(df):
+            self._validate_time_series(df)
+
+        cats = [
```

Comment:
is the code here duplicating `self._validate_time_series`?

Reply:
I refactored into `_validate_order_of_variables`, but forgot to delete here. Will do that now